director/urls.py

```diff
@@ -58,11 +57,6 @@
     # API
     path(UrlRoot.api.value + '/', include('api_urls')),
 
-    # Checkouts App
-    path(UrlRoot.checkouts.value + '/', include('checkouts.urls')),
```

Comment:
Please remove these definitions from the UrlRoot.checkouts Enum if they are no longer used.

Reply:
I resolve this conversation by reading the diff better 😂